jumanji/environments/logic/rubiks_cube/env.py

```diff
@@ -211,11 +174,6 @@ def step(
         )
         return next_state, next_timestep
 
-    def get_action_history(self, state: State) -> chex.Array:
```

Comment:
Would we like to keep the notion of `action_history` (or rather sequence of actions used to scramble) in the `ScramblingGenerator` instead of removing it completely?

Reply:
It is accessible by following through what the generator does, but I thought we agreed not to explicitly store it anywhere?

Reply:
You mean the sequence of actions used to reset is accessible by calling `generator.generate_actions_for_scramble(key)` on the same key used for reset? Then, I'm happy with this and we can resolve this.

Reply:
Correct